mesheryctl/internal/cli/root/credentials/credential_test.go

```diff
@@ -19,6 +19,7 @@ func TestCredentialCmd(t *testing.T) {
 	// initialize mock server for handling requests
 	utils.StartMockery(t)
 
+	go utils.StartMockMesheryServer(t)
 	// create a test helper
```

Comment:
@singh1203 there are a couple of lint checks to review.

Reply:
this check is resolved but there is another from the master branch due to which `Golang Unit and Integration Tests / Unit tests` is failing